manifests/bundle.pp

```diff
@@ -87,7 +87,7 @@
   }
 
   if $multicore {
-    if $multicore == '0' or $multicore >= $::processorcount {
+    if $multicore == '0' or versioncmp($multicore,$::processorcount) >= 0 {
```

Comment:
bastardization of a function due to the lack of decent string/integer comparisons.